### ISSUE TITLE
UX Guide Order of Information - Audiobooks

### ISSUE THREAD

**Issue by wareid** [w3c/publ-a11y#28]
In reviewing the UX Guide for Displaying Accessibility Metadata, I had one question about the order of key information in regards to audiobooks. 

I think knowing whether content has audio is really important, but I think placing audiobooks in this order might be incongruent with how content metadata works in practice, and also a little late in the experience for a user to find out if the content is audio. 

I think emphasizing the content type (EPUB, audiobook, etc.) should be elevated into the primary metadata (like in section 3, example 2), with the accessibility information containing details like whether there is a transcript, media overlays, or other features. 

Alternately, audiobook could be used to tell users that an audio version of the book is available, but I think it's awkward for users to only find out something is an audiobook in a sub-section. In practice, it is probably unlikely and potentially repetitive to have the information there as well.

**Comment by avneeshsingh** [comment]
Example 2 in Section 3 is not part of formal guidelines. It is more to give an example for placing link to the displayed accessibility metadata.
Looking from the context of accessibility information, if audio books need to be there, then I would say having screen reader friendly is even more important to elevate because most of the publications in the main stream are text only publications, and some of us may say that highlighting Hazards is of paramount importance. Therefore it would be better to avoid going into this detail.

I do understand that placing information related to format is useful in the examples provided in section 3 for example, if the publication is EPUB, a PDF, an audio book etc. I think it can address the issue of identifying audio books from the general metadata page.

**Comment by avneeshsingh** [comment]
In our recent con call we decided to rename audio book to full audio. It will eliminate the confusion with the term audio book, which has a specific public perception.
Term full audio is focus on accessibility, and includes audio books as well as audio sync with text. This also makes it more clear that this new term should be on accessibility page instead of general metadata page.
